package.json

```diff
@@ -17,15 +17,14 @@
   },
   "homepage": "https://github.com/microsoft/vscode-zeromq#readme",
   "scripts": {
-    "test": "node ./lib/index.test.js"
+    "test": "node ./lib/index.test.js",
+    "downloadBinaries": "node ./build/prePublish.js"
   },
-  "dependencies": {
+  "devDependencies": {
+    "@types/node": "^18.15.0",
     "https-proxy-agent": "^5.0.1",
```

Comment:
Now that downloading of binaries is done dueing publishing, these dependencies are only required for dev time,